cli/test/smokehouse/test-definitions/dobetterweb.js

```diff
@@ -585,6 +585,13 @@ const expectations = {
           },
         },
       },
+      'metrics': {
+        details: {items: {0: {
+          timeToFirstByte: '450+/-100',
+          lcpLoadStart: '7750+/-500',
+          lcpLoadEnd: '7750+/-500',
```

Comment:
I assume these are pretty stable if you're adding them? That's great, I was worried we wouldn't have any viable smoke testing for these

Is it possible to move lcpLoadStart/lcpLoadEnd apart or does the image load so fast their possible timings overlap too much?

Reply:
I did have another test that was too flaky, I think this needs to be tested in isolation.

> Is it possible to move lcpLoadStart/lcpLoadEnd apart or does the image load so fast their possible timings overlap too much?

This

Reply:
Ah, yeah. I wonder if there's something we could do with the static-server `delay` headers, but we could wait for when the timings land as a visible audit in the report